modules/cco-ccoctl-deleting-sts-resources.adoc

```diff
@@ -48,6 +48,11 @@ where:
 +
 ** `<name>` matches the name used to originally create and tag the cloud resources.
 ** `<aws_region>` is the AWS region in which cloud resources will be deleted.
++
+[NOTE]
+====
+If your cluster uses Technology Preview features that are enabled by the `TechPreviewNoUpgrade` feature set, you must include the `--enable-tech-preview` parameter.
+====
```

Comment:
`ccoctl aws delete` does deletion based on name tags assigned to cloud resources. It does not depend on extracted credentials requests manifests for identifying resources. So we can remove this section here.

Reply:
Removed for AWS, as well as GCP since it works the same way.